### ISSUE TITLE
[Suggestion] Some remarks on the tutorial zone, and some QoL gameplay suggestions

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2128]
I'm writing these as I go through the tutorial and whilst it's pretty comprehensive and shows of good level design skills, I'd like to share some tips to hopefully make it better:

1. Since I like to look into every nook and cranny, I get annoyed by constant pop-up windows I have to manually close. Please make them all HUD messages, so they are less obtrusive.
3. The elevator taking you up to the Zone 2 could use a recall button in case you fall off the platform. I know you can just use the anti-gravity function but it'd be more immersive that way.
4. Let the player know that they need to scroll through the hotbar in order to pick new items instead of replacing the anti-grav tool.
5. In Zone 2 the tutorial instructs you to rotate the block using the [R] key. Instead, it's [Q] and [E] keys.
6. Provide the Capacitor block as a pickable item, the same way the Reactor and previous blocks were provided.
7. The auto turret section was a bit vague. Would be helpful if the tutorial demonstrated a practical application of the AI turret by shooting a hostile (stationary and unarmed) target.

I've got some general gameplay suggestions, too:

A. Make the HUD messages scroll down way slower so people can have enough time to read them.
B. Let player characters take a seat and use the yoke as soon as the loading circle (when you're holding [F]) completely fills up - the same way you can stop using the yoke and exit the seat instantly once the loading circle is full.

**Comment by tsunamayo** [comment]
Thanks for the suggestion, they are all very valid.
Once thing is I didnt understood the last one (B) with the seat. You mean once the circle is full is transition is done? Thanks

**Comment by Kaiser-Indrasil** [comment]
Yeah - notice how your character leaves the seat and yoke as soon as circle is full (not waiting for you to release the [F] key) while the entering the seat and using the yoke actions need you to release the [F] key before the action is executed.

**Comment by tsunamayo** [comment]
okay indeed, it is a key release timing. I keep you posted.
